src/components/InternalTransferCell.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <span v-if="activity.payload?.kind !== 'reward_block'">N/A</span>
+  <template v-else>
+    <app-chip size="sm">
+      {{ formatAePrice(formatAettosToAe(activity.payload.amount)) }}
+    </app-chip>
+  </template>
```

Comment:
```suggestion
 <template v-if="activity.payload?.kind !== 'reward_block'">
    N/A
  </template>
  <app-chip
    v-else
    size="sm">
    {{ formatAePrice(formatAettosToAe(activity.payload.amount)) }}
  </app-chip>
```

I think it can be achieved with less code

Reply:
Oh, right, of course!